components/efuse/src/esp_efuse_utility.c

```diff
@@ -452,11 +461,29 @@ bool esp_efuse_utility_load_efuses_from_flash(void)
             return true;
         }
     }
+
+    ESP_EARLY_LOGW(TAG, "no fuses found in flash");
+
     return false;
 }
 
+void write_efuses_to_ram(void)
+{
+    ESP_EARLY_LOGW(TAG, "Writing fuses to ram");
+
+    memset(virt_blocks, 0, sizeof(virt_blocks));
+
+    esp_efuse_utility_debug_dump_blocks();
+
+    ESP_EARLY_LOGW(TAG, "Write complete");
```

Comment:
I suggest keeping this function quiet. In this func, we just reset efuses. I think it is ok to keep the original approach.